src/CTA.WebForms2Blazor/ClassConverters/ControlCodeBehindClassConverter.cs

```diff
@@ -14,22 +15,26 @@ namespace CTA.WebForms2Blazor.ClassConverters
 {
     public class ControlCodeBehindClassConverter : ClassConverter
     {
+        private const string ActionName = "ControlCodeBehindClassConverter";
```

Comment:
We can improve maintenance and repetition by adding `ActionName` as a property to the base class `ClassConverter` and defining it as:

```csharp
public class ClassConverter
{
    protected virtual string ActionName => GetType().Name;
    ...
}
```

This can be done for all converter base classes

Reply:
I was thinking that as well, only reason I used the constants is if someone don't name the class name properly. I will make the change